lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -101,6 +107,7 @@ type VicContainerProxy interface {
 
 	Stop(vc *viccontainer.VicContainer, name string, seconds *int, unbound bool) error
 	State(vc *viccontainer.VicContainer) (*types.ContainerState, error)
+	GetStateFromHandle(op trace.Operation, handle string) (string, string, error)
```

Comment:
We might want to have this reflect the same state that is returned by either GetContainerInfoHandler or in the GetContainerListHandler - those all pull from Runtime and Handle state, it's just that it's pulled from the exec cache instead of from the provided handle.